sdk/python/feast/stream_feature_view.py

```diff
@@ -119,7 +119,7 @@ def __init__(
         super().__init__(
             name=name,
             entities=entities,
-            ttl=ttl or timedelta(days=0),
```

Comment:
may be better to have these changes in a different pr but won't block you

Reply:
yup will make separate PR